compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -219,6 +220,23 @@ getVectorPreProcStrategy(linalg::LinalgOp linalgOp) {
     return VectorPreProcStrategy::None;
   }
 
+  // Walk the linalgOp code, if there is any instruction that could result in
+  // undefined behavior in mask strategy, fall back to using peel strategy.
+  bool usePeelingStrategy = false;
+  linalgOp.walk([&](Operation *op) -> WalkResult {
+    if (isa<mlir::arith::RemSIOp, mlir::arith::RemUIOp, mlir::arith::DivSIOp,
+            mlir::arith::DivUIOp, mlir::arith::CeilDivSIOp,
+            mlir::arith::CeilDivUIOp, mlir::arith::FloorDivSIOp,
+            mlir::arith::DivFOp, mlir::arith::RemFOp>(op)) {
+      usePeelingStrategy = true;
+      return WalkResult::interrupt();
+    }
+    return WalkResult::advance();
+  });
```

Comment:
I think it's better to create a method for it. You can put it to `LLVMCPU/Utils.[h|cpp]`: https://github.com/iree-org/iree/blob/main/compiler/src/iree/compiler/Codegen/LLVMCPU/Utils.h#L53

Reply:
Done.